exporters/common/src/main/java/io/opentelemetry/exporter/internal/http/HttpExporterBuilder.java

```diff
@@ -125,17 +129,27 @@ public HttpExporter<T> build() {
     Map<String, String> headers = this.headers == null ? Collections.emptyMap() : this.headers;
     Supplier<Map<String, String>> headerSupplier = () -> headers;
 
-    HttpSender httpSender =
-        new OkHttpHttpSender(
-            endpoint,
-            compressionEnabled,
-            exportAsJson ? "application/json" : "application/x-protobuf",
-            timeoutNanos,
-            headerSupplier,
-            authenticator,
-            retryPolicy,
-            tlsConfigHelper.getSslContext(),
-            tlsConfigHelper.getTrustManager());
+    HttpSender httpSender = null;
+    for (HttpSenderProvider httpSenderProvider :
```

Comment:
Is there a case for more than one sender provider?
Should we force to have just one? I think in most cases, having 2 of them is more of a bug due to conflicting dependencies.

Reply:
Could we perhaps print a warning when >1 sender implementations are found on classpath?

Reply:
That's fine, but the question remains. 
> Is there a case for more than one sender provider?

Reply:
This is reasonable to me.

Reply:
Added a TODO for now which I'll resolve in the next PR.